packages/jest-cli/src/TestRunner.js

```diff
@@ -123,43 +127,9 @@ class TestRunner {
 
   runTests(testPaths: Array<string>, maybeReporter?: TestReporter) {
```

Comment:
You removed support for `maybeReporter` but the argument is still there. We also rely on this in our internal runner (ugh!) so we need to make a bunch of changes once we sync. Basically in www we inject the istanbulreporter here, but I'm not sure why we do that as its the default anyway.